phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ApplicationSideBar.java

```diff
@@ -158,7 +158,7 @@ private ToggleButton createAllCategoriesToggleButton() {
     private ToggleButton createCategoryToggleButton(CategoryDTO category) {
         final LeftToggleButton categoryButton = new LeftToggleButton(category.getName());
 
-        categoryButton.setId(String.format("%sButton", category.getName().toLowerCase()));
+        categoryButton.setId(String.format("%sButton", category.getId().toLowerCase()));
```

Comment:
If this is the case, I think you should watch out when you use ids with ` ` spaces in the tests. 
I think you should ensure here that the id contains no space characters or other "bad" symbols, right?